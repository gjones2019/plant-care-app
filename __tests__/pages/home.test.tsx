import { render, screen } from '@testing-library/react'
import Home from '../../pages/index'

describe('Home', () => {
  it('renders heading', () => {
    render(<Home />)

    const heading = screen.getByRole('heading', {
      name: "Welcome to My Plant Care App",
    })

    expect(heading).toBeInTheDocument()
  })

  it('renders home page unchanged', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
  })
})