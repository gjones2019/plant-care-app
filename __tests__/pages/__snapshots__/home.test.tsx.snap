// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders home page unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="centeredContent"
    >
      <h1
        class="heading"
      >
        Welcome to My Plant Care App
      </h1>
      <div
        class="main"
      >
        <div
          class="grid"
        />
      </div>
    </div>
  </div>
</div>
`;
